.drone.yml

```diff
@@ -236,3 +253,14 @@ data:
       trigger:
         event:
           - pull_request
+    - name: notification-service-lint
+      type: kubernetes
+      depends_on: [publish-notification-service-on-pr-push]
+      image: ghcr.io/sidestream-tech/unified-auctions-ui/notification-service:pr-${DRONE_PULL_REQUEST}
+      workdir: /notification-service
+      steps_parallel:
+        - name: lint
+          command: npm run lint
```

Comment:
Do you also want to build it to validate types?

Reply:
Does it not get built in this step? 

https://github.com/sidestream-tech/unified-auctions-ui/blob/6e76a858f1ddecb393a12b130d24d40fae9ed556/.drone.yml#L158

Reply:
Good point, I guess it does, I hope it also results in CI error if it fails though